addons/voice/org.openhab.voice.googletts/src/main/java/org/openhab/voice/googletts/internal/GoogleTTSService.java

```diff
@@ -107,7 +106,7 @@
      */
     @Activate
     protected void activate(Map<String, Object> config) {
-        //create home folder
+        // create home folder
```

Comment:
One general question here: I saw this in my logs:
```
2018-11-18 11:16:33.318 [INFO ] [.googletts.internal.GoogleTTSService] - Using home folder: /var/lib/openhab2/googletts
```
But it looks like we never use this folder. There is not data in it. We only create it. Is there a specific reason for that?

Maybe @gbicskei can comment on this too.

Reply:
I've removed it since it is unused nowadays. My best guess is that the Google Cloud JSON was read from it initially.